### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by ThirtyThreeB**

### CODE REVIEW

File: lib/calc_web/controllers/page_controller.ex [hunk 1]
```diff
@@ -1,7 +1,17 @@
 defmodule CalcWeb.PageController do
   use CalcWeb, :controller
 
-  def index(conn, _params) do
-    render(conn, "index.html")
+  def index(conn, params) do
+
+    if (params |> is_nil()) do
+      conn
+      |> put_flash(:info, "")
```
Comment (taddeimania) [inline lib/calc_web/controllers/page_controller.ex:8]: you don't need to put a flash if you aren't putting anything into it

File: lib/calc_web/templates/page/index.html.heex [hunk 2]
```diff
@@ -1,41 +1,34 @@
...
-      <li>
-        <a href="https://discord.gg/elixir">Elixir on Discord</a>
-      </li>
-    </ul>
-  </article>
-</section>
+<div class="card mt-5" style="width: 48rem;">
+  <div class="card-body">
+    <h5 class="card-title mb-4">Craptacular Calculator</h5>
+    <form>
```
Comment (taddeimania) [inline lib/calc_web/templates/page/index.html.heex:5]: think about how you'd rework this to use a `form_for` and the phoenix fields

File: mix.exs [hunk 3]
```diff
@@ -40,13 +40,13 @@ defmodule Calc.MixProject do
       {:floki, ">= 0.30.0", only: :test},
       {:phoenix_live_dashboard, "~> 0.6"},
       {:esbuild, "~> 0.3", runtime: Mix.env() == :dev},
+      {:dart_sass, "~> 0.2", runtime: Mix.env() == :dev},
```
Comment (taddeimania) [inline mix.exs:43]: whats all this about?

File: lib/calc_web/controllers/page_controller.ex [hunk 4]
```diff
@@ -1,7 +1,17 @@
...
+      |> render("index.html")
+    else
+      result = Operations.operate(params)
```
Comment (taddeimania) [inline lib/calc_web/controllers/page_controller.ex:11]: where is `Operations.operate/1` defined?

File: mix.exs (hunk 3 above)
Comment (ThirtyThreeB) [reply mix.exs:43]: had to do this as part of getting bootstrap running

File: lib/calc_web/controllers/page_controller.ex (hunk 4 above)
Comment (ThirtyThreeB) [reply lib/calc_web/controllers/page_controller.ex:11]: I used `git commit -am` which now that I remember, doesn't add untracked files.  Thanks Linus

**Pull request merged**